### DISCUSSION TITLE
Missing QSOs with Station Locations

### DISCUSSION THREAD

**Discussion by Cadair** [magicbug/Cloudlog#2437] [Q&A] [answered]
Hi all,

I was an early tester of the station locations branch, and it seems that I have got my cloudlog into a situation where it's a little confused.
 
Not all my QSOs show up in all the places I expect. For instance on the Station Locations page:

![2023-08-26-175723_grim](https://github.com/magicbug/Cloudlog/assets/1391051/d1037eeb-51c5-4b15-8edc-cbf7bdf993b9)

Where as under my Station Logbook I have a list of 12 linked locations.

I am very confused by what the situation is supposed to be, what I have and what might have gone wrong. Any pointers would be very welcome, thanks.

Edit: One other thing, under "advanced search" I can only see the 16 QSOs listed under the station with ID 13. When I would like to search all of the possible QSOs.

**Comment by magicbug** [comment, marked as answer]
I'm going to go out on a limb and say that has probably happened is you have two cloudlog users and the one your using doesn't have the Station Locations associated with it.

Quickest fix would be use a tool like phpmyadmin and go and adjust the Cloudlog database in this case the station_profile table and make sure the user_id column matches for each one

**Reply by Cadair** [reply]
Thanks a lot for the quick response, I will open up the DB and have a look. I don't think that's likely to be the issue tho as I think I have only ever had one user on this deployment.

**Reply by Cadair** [reply]
ok you were completely correct, there are two users and they are all associated with the other one. I shall just modify the db.

**Reply by Cadair** [reply]
This worked, thanks a lot :)
